### ISSUE TITLE
Create baseline

### ISSUE THREAD

**Issue by dwinzg** [NyxHuang96/Fast-Repository-for-Analyzing-Unreliable-Discourse#86]
Baselines

rubric={quality:2, efficiency:2, reasoning:2}

You are now pretty good at building machine-learning pipelines,
and have a glut of tools at your disposal. Use them!

We create a baseline using a simple model (remember MVPs?), training it on our train set,
and evaluating on our dev set
(test sets don't get touched until after everything is implemented - it's too easy to
overfit to our test set, if we're not careful...)

Create a data processing, training, and evaluation pipeline.

You should pick two methods to train (one neural, one traditional?), and establish a
training pipeline. You are free to use any of the tools we've talked about this year
(or others you know), except for LLMs / AI. We'll get to those in later sprints...

Evaluate and report your results on your dev set. It may help your report if you also
do a more in-depth analysis of the types of errors your model is making
(including qualitative analysis).

Document everything.
